skyVegasTests/stakeUpdateGameClient.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, type Page } from '@playwright/test';
+import {
+    launchGame,
+    login,
+    validateConsoleMessages,
+    startEventListener,
+    readGames,
+} from './utilities';
+import { ExpectedMessage } from '../types/expectedMessage';
+
+let page: Page;
+readGames().forEach((game) => {
+    const consoleMessages: string[] = [];
+
+    test.describe(`Testing with text: ${game}`, async () => {
+        test.beforeAll(async ({ browser }) => {
+            page = await browser.newPage();
+            await login(page);
+        });
+
+        test.afterAll(async () => {
+            await page.close();
+        });
+
+        test('stakeUpdate', async () => {
+            startEventListener(page, consoleMessages);
+            await launchGame(page, game, consoleMessages);
+
+            await page.mouse.click(300, 300, { delay: 10 });
+            await page.keyboard.press('Enter', { delay: 2000 });
+            await page.mouse.click(300, 300, { delay: 10 });
+            await page.keyboard.press('ArrowRight', { delay: 2000 });
+            await validateConsoleMessages(
```

Comment:
Is validate console messages needed? There will always be a stake update message because one comes through when launching the game. It won't therefore validate whether we have changed the stake.

Reply:
That's why I put the lines:

const lastMessage = consoleMessages[consoleMessages.length - 1];
test.expect(lastMessage).toContain(ExpectedMessage.STAKE_UPDATE);

So this checks the last message in the console, this will then validate that we changed the stake. But I think validateConsoleMessages function already checked that, so it's been removed.